README.md

```diff
@@ -1,64 +1,82 @@
-## About
+## Open Source Routing Machine
 
-The Open Source Routing Machine is a high performance routing engine written in C++11 designed to run on OpenStreetMap data.
+[![Travis](https://travis-ci.org/Project-OSRM/osrm-backend.png?branch=master)](https://travis-ci.org/Project-OSRM/osrm-backend)
+[![AppVeyor](https://ci.appveyor.com/api/projects/status/4iuo3s9gxprmcjjh)](https://ci.appveyor.com/project/DennisOSRM/osrm-backend)
+[![Codecov](https://codecov.io/gh/Project-OSRM/osrm-backend/branch/master/graph/badge.svg)](https://codecov.io/gh/Project-OSRM/osrm-backend)
 
-## Current build status
+High performance routing engine written in C++14 designed to run on OpenStreetMap data.
 
-| build config | status |
-|:-------------|:-------|
-| Linux        | [![Build Status](https://travis-ci.org/Project-OSRM/osrm-backend.png?branch=master)](https://travis-ci.org/Project-OSRM/osrm-backend) |
-| Windows      | [![Build status](https://ci.appveyor.com/api/projects/status/4iuo3s9gxprmcjjh)](https://ci.appveyor.com/project/DennisOSRM/osrm-backend) |
-| Coverage     | [![codecov](https://codecov.io/gh/Project-OSRM/osrm-backend/branch/master/graph/badge.svg)](https://codecov.io/gh/Project-OSRM/osrm-backend) |
+The following services are available via HTTP API, C++ library interface and NodeJs wrapper:
+- Nearest - Snaps coordinates to the street network and returns the nearest matches
+- Route - Finds the fastest route between coordinates
+- Table - Computes the duration of the fastest route between all pairs of supplied coordinates
+- Match - Snaps noisy GPS traces to the road network in the most plausible way
+- Trip - Solves the Traveling Salesman Problem using a greedy heuristic
+- Tile - Generates Mapbox Vector Tiles with internal routing metadata
+
+To quickly try OSRM use our [free and daily updated demo server](http://map.project-osrm.org).
```

Comment:
given that that thing hasn't been updated for more than 3 weeks now, I would feel better if we don't claim `daily` here.